bus_schedule/bus_schedule_esp32/bus_schedule_esp32.ino

```diff
@@ -164,6 +172,15 @@ void handleResponse(String json, uint8_t stopIndex) {
     putString(terminal, data->terminal, sizeof(data->terminal));
     putString(departure, data->departure, sizeof(data->departure));
 
+    data->scheduleRelationship = ScheduleRelationship::UNKNOWN;
```

Comment:
Is it worth keeping all the possible options?

You only use `skipped ?`, so you could just have a boolean `scheduled` in `struct BusData`.